### ISSUE TITLE
what is JCR

### ISSUE THREAD

**Issue by vphill** [unt-libraries/JCR#1]
I'd unpack the acronym somewhere in the readme.

**Comment by ChrisHolladay** [comment]
Absolutely, I'll break it down a little more in the readme. It's the query and design code base for a talk that we gave at CTLC last Friday.
